### DISCUSSION TITLE
How can I Test Airspy Sample rate on Ubuntu?

### DISCUSSION THREAD

**Discussion by pmaine** [AlexandreRouma/SDRPlusPlus#319] [Q&A] [answered]
The airspy website quickstart guide provides a way to test the max sample rate before packets are dropped. How can I make a similar test on an intel Ubuntu system?

**Comment by the-gobleo-tinkerer** [comment, marked as answer]
I looked through the quick start guide but I'm not very sure I found the thing you were referring to. If you meant the command `airspy_rx -r NUL -t 0` and its related steps, you can do the same on Linux by replacing `NUL` with `/dev/null`.

**Reply by pmaine** [reply]
That worked! I did get an average of 10MSPS!
Thank you

**Reply by bvernoux** [reply]
Such questions are answered on https://github.com/airspy/airspyone_host/wiki/Troubleshooting

**Reply by the-gobleo-tinkerer** [reply]
Thanks @bvernoux for the link.
